src/app/cluster/duck/operations.ts

```diff
@@ -76,19 +79,70 @@ const addCluster = clusterValues => {
   };
 };
 
+const updateCluster = clusterValues => {
+  return async (dispatch, getState) => {
+    try {
+      const state = getState();
+      const { migMeta } = state;
+      const client: IClusterClient = ClientFactory.hostCluster(state);
+
+      const clusterReg = updateClusterRegistryObj(
+        clusterValues.url
+      );
+      const tokenSecret = updateTokenSecret(
+        clusterValues.token
+      );
+      const clusterRegResource = new ClusterRegistryResource(
+        ClusterRegistryResourceKind.Cluster,
+        migMeta.namespace
+      );
+      const secretResource = new CoreNamespacedResource(
+        CoreNamespacedResourceKind.Secret,
+        migMeta.configNamespace
+      );
+
+      const arr = await Promise.all([
+        client.patch(clusterRegResource, clusterValues.name, clusterReg),
+        client.patch(secretResource, clusterValues.name, tokenSecret),
+      ]);
+      dispatch(updateClusterSuccess);
```

Comment:
@ibolton336 please take a look to see if I've updating this as you expected....I feel like I'm not doing this correctly.

I haven't created any function to handle the update/remove success.... for either clusters or storage, not sure if there is some magic creating this or if I need to define it.

Reply:
Addressed